pkg/config/config.go

```diff
@@ -345,7 +346,15 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("dogstatsd_entity_id_precedence", false)
 	// Sends Dogstatsd parse errors to the Debug level instead of the Error level
 	config.BindEnvAndSetDefault("dogstatsd_disable_verbose_logs", false)
-	config.SetKnown("dogstatsd_mapper_profiles")
+
+	config.BindEnv("dogstatsd_mapper_profiles") //nolint:errcheck
```

Comment:
`//nolint:errcheck` is not intent to be added in a new PR. `//nolint:errcheck` means check how to handle the error.